backend/src/test/java/com/woowacourse/gongcheck/core/application/JobServiceTest.java

```diff
@@ -65,272 +70,518 @@ class JobServiceTest {
-            Long updateJobId = jobService.updateJob(host.getId(), savedJobId, jobCreateRequest);
-
-            assertThat(updateJobId).isNotNull();
+        @Nested
+        class host가_존재하지_않는_경우 {
+
+            private JobCreateRequest request;
+            private Long savedJobId;
+
+            @BeforeEach
+            void setUp() {
+                Host host = hostRepository.save(Host_생성("1234", 1234L));
+                Space space = spaceRepository.save(Space_생성(host, "잠실"));
+                List<TaskCreateRequest> tasks = List.of(new TaskCreateRequest("책상 닦기"), new TaskCreateRequest("칠판 닦기"));
+                List<SectionCreateRequest> sections = List.of(new SectionCreateRequest("대강의실", tasks));
+                request = new JobCreateRequest("청소", sections);
+                savedJobId = jobService.createJob(host.getId(), space.getId(), request);
+            }
+
+            @Test
+            void 예외가_발생한다() {
+                assertThatThrownBy(() -> jobService.updateJob(0L, savedJobId, request))
+                        .isInstanceOf(NotFoundException.class)
+                        .hasMessage("존재하지 않는 호스트입니다.");
+            }
         }
 
-        @Test
-        void host가_존재하지_않을_경우_예외가_발생한다() {
-            assertThatThrownBy(() -> jobService.updateJob(0L, savedJobId, jobCreateRequest))
-                    .isInstanceOf(NotFoundException.class)
-                    .hasMessage("존재하지 않는 호스트입니다.");
+        @Nested
+        class host에_해당하는_jobId가_아닐_경우 {
+
+            private JobCreateRequest request;
+            private Long savedJobId;
+
+            @BeforeEach
+            void setUp() {
+                Host host = hostRepository.save(Host_생성("1234", 1234L));
+                Space space = spaceRepository.save(Space_생성(host, "잠실"));
+                List<TaskCreateRequest> tasks = List.of(new TaskCreateRequest("책상 닦기"), new TaskCreateRequest("칠판 닦기"));
+                List<SectionCreateRequest> sections = List.of(new SectionCreateRequest("대강의실", tasks));
+                request = new JobCreateRequest("청소", sections);
+                savedJobId = jobService.createJob(host.getId(), space.getId(), request);
+            }
+
+            @Test
+            void 예외가_발생한다() {
+                Host anotherHost = hostRepository.save(Host_생성("1234", 2345L));
+                assertThatThrownBy(() -> jobService.updateJob(anotherHost.getId(), savedJobId, request))
+                        .isInstanceOf(NotFoundException.class)
+                        .hasMessage("존재하지 않는 작업입니다.");
+            }
         }
 
-        @Test
-        void host에_해당하는_jobId가_아닐_경우_예외가_발생한다() {
-            Host anotherHost = hostRepository.save(Host_생성("1234", 2345L));
-            assertThatThrownBy(() -> jobService.updateJob(anotherHost.getId(), savedJobId, jobCreateRequest))
-                    .isInstanceOf(NotFoundException.class)
-                    .hasMessage("존재하지 않는 작업입니다.");
+        @Nested
+        class 존재하지_않는_Job을_수정할_경우 {
+
```

Comment:
```suggestion
        @Nested
        class 입력받은_Job_id가_존재하지_않는_경우 {

```
이렇게 하면 테스트 전체를 읽을 때 더 자연스럽지 않을까요?

Reply:
"존재하지 않는 job id를 입력받을 경우"는요?.?